bio.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Vacation planner</title>
+</head>
+
+<body>
+    <a href="index.html">Home |</a>
+    <a href="bio.html">Bio |</a>
+    <a href="schedule.html">Schedule</a>
+    <p></p>
+    <img src="https://picsum.photos/id/1084/536/354?grayscale" width="300" hight="200" />
```

Comment:
In marea majoritate a cazurilor nu vrem sa cream pagini care contin elemente care nu au continut (in cazul de fara tag-ul <p> nu afiseaza nimic)
Daca vrei sa afisezi imaginea sub link-uri, ce ai putea face e sa curpinzi link-urile intr-un tag de nav ori sa adaugi imaginea intr-un div. Aceste elemente sunt block tags care ocupa intregul width al paginii (https://www.w3schools.com/html/html_blocks.asp);

```suggestion
    <nav>
        <a href="index.html">Home |</a>
        <a href="bio.html">Bio |</a>
        <a href="schedule.html">Schedule</a>
     </nav>
     <img src="https://picsum.photos/id/1084/536/354?grayscale" width="300" hight="200" />
```